bin/rialto/node/Cargo.toml

```diff
@@ -1,3 +1,11 @@
+[[bin]]
```

Comment:
It doesn't break our test deployments, right? I mean - do we nee those binaries for relay/parachain to work? Do we still able to build that good old `rialto-bridge-node`, which has many references in our code?

Reply:
Yes, it works. Tested locally.

We need these binaries. Otherwise the parachain won't produce blocks,